src/components/Button.js

```diff
@@ -0,0 +1,37 @@
+import styled, { css } from 'styled-components';
+
+export default function Button() {
+  return (
+    <ButtonStyled aria-label="button" role="button">
+      Create a card
+    </ButtonStyled>
+  );
+}
```

Comment:
I think you don´t really need the function here. You could do: 
```suggestion
export default styled.button`
....
`
```
...directly and import the Button component where you need it :)